### ISSUE TITLE
set item use two grid

### ISSUE THREAD

**Issue by mxyue** [fyne-io/fyne#623]
``` go
widget.NewVBox(
  fyne.NewContainerWithLayout(layout.NewGridLayout(3),
    widget.NewButton("button use 1 grid", func() {}),
    widget.NewLabel("I want use two grid")),
)
```
I not found some method can set rowspan。 Is there some method to set the width ?

**Comment by andydotxyz** [comment]
This is not currently possible sorry - but it is a good feature request.

**Comment by andydotxyz** [comment]
I just realised this would be addressed by the requested flexbox layout #47, closing as duplicate
